### ISSUE TITLE
zeroconf problem with zeroconf_type = "_%s._tcp." % __appname__

### ISSUE THREAD

**Issue by stephane-martin** [nicolargo/glances#888]
Recent versions of the zeroconf python package doesnt like a zeroconf type that ends with '._tcp.'.

It triggers an exception when trying `glances --browser`.

I had to patch this way:
```
--- a/glances/autodiscover.py
+++ b/glances/autodiscover.py
@@ -47,7 +47,7 @@
         sys.exit(1)

 # Global var
-zeroconf_type = "_%s._tcp." % __appname__
+zeroconf_type = "_%s._tcp.local." % __appname__


 class AutoDiscovered(object):
```

**Comment by nicolargo** [comment]
Confirmed with ZeroConf 0.17.4:

```
$  python -m glances -s
Traceback (most recent call last):
  File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/nicolargo/Dropbox/dev/glances/glances/__main__.py", line 29, in <module>
    glances.main()
  File "glances/__init__.py", line 174, in main
    args=args)
  File "glances/server.py", line 210, in __init__
    self.autodiscover_client = GlancesAutoDiscoverClient(socket.gethostname(), args)
  File "glances/autodiscover.py", line 210, in __init__
    weight=0, priority=0, properties={}, server=hostname)
  File "/usr/local/lib/python2.7/dist-packages/zeroconf.py", line 1191, in __init__
    if not type_.endswith(service_type_name(name)):
  File "/usr/local/lib/python2.7/dist-packages/zeroconf.py", line 203, in service_type_name
    "Type must end with '._tcp.local.' or '._udp.local.'")
zeroconf.BadTypeInNameException: Type must end with '._tcp.local.' or '._udp.local.'
```

**Comment by nicolargo** [comment]
Corrected in the develop branch, will be available in Glances 2.7.

Thanks for the patch (again) !
